### DISCUSSION TITLE
List of all CAIP-2 supported chains in wallet connect

### DISCUSSION THREAD

**Discussion by vitoff34** [WalletConnect/walletconnect-monorepo#3527] [Developer Support] [answered]
Hello, 
where can I find a list of all CAIP-2 namespaces for all supported chains in wallet connect. In the docs I can find examples of only some cosmos and solana chains.
Thank you

**Comment by bumblefudge** [comment, marked as answer]
https://docs.walletconnect.com/2.0/advanced/multichain/chain-list
^ This is all known/registered chains, although technically what a consenting dapp and a consenting wallet do in the privacy of their own homes is up to them (particularly if they use the namespaces/CAIP-25 protocol properly).  For example, private or permissioned networks using a valid chainID in any of the following namespaces could also be used...
https://namespaces.chainagnostic.org/

**Reply by vitoff34** [reply]
Thank you very much! https://namespaces.chainagnostic.org/ this is what I was looking for
